factory/mock/singlesignerMock.go

```diff
@@ -11,12 +11,12 @@ import (
 type SinglesignMock struct {
 }
 
-// Sign Signs a message with using a single signature schnorr scheme
+// Sign signs a message using a single signature Schnorr scheme
```

Comment:
Might remove the "Schnorr scheme" from all method names. Unrelated

Reply:
done